euphoria-examples/build.gradle

```diff
@@ -1,8 +1,35 @@
+plugins {
+  id 'java'
+  id 'com.github.johnrengelman.shadow'
```

Comment:
is this really needed here? java plugin is there by default and shadow plugin should be just applied

```
apply plugin: 'com.github.johnrengelman.shadow'
```

Reply:
:+1: